docs/cn/sql-reference/20-sql-functions/07-aggregate-functions/aggregate-histogram.md

```diff
@@ -0,0 +1,121 @@
+---
+title: HISTOGRAM
+---
+
+Aggregate function.
+
+The HISTOGRAM() is used to describe the distribution of the data. It uses an "equal height" bucking strategy to generate the histogram. The result of the function returns an empty or Json string.
+
+## Syntax
+
+```sql
+HISTOGRAM(<expr>)
+HISTOGRAM(<expr> [, max_num_buckets])
+```
```

Comment:
It would better to explain how the func decide the num of buckets when max_num_buckets is not explicitly specified.